test/jasmine/tests/annotations_test.js

```diff
@@ -505,7 +510,7 @@ describe('annotations log/linear axis changes', function() {
         var mockData = Lib.extendDeep([], mock.data);
         var mockLayout = Lib.extendDeep({}, mock.layout);
 
-        Plotly.plot(gd, mockData, mockLayout).then(done);
+        return Plotly.plot(gd, mockData, mockLayout).then(done);
```

Comment:
This `return` is unnecessary - `.then(done)` is all it needs.

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/5340/commits/97574f966612ce1d6b669a35c10c356e09c8da7a.